src/fsharp/Optimizer.fs

```diff
@@ -1252,9 +1252,12 @@ let AbstractAndRemapModulInfo msg g m (repackage, hidden) info =
 // Misc helpers
 //------------------------------------------------------------------------- 
 
-// Mark some variables (the ones we introduce via abstractBigTargets) as don't-eliminate 
+/// Mark some variables (the ones we introduce via abstractBigTargets) as don't-eliminate 
 let [<Literal>] suffixForVariablesThatMayNotBeEliminated = "$cont"
 
+/// Indicates a ValRef generated to facilitate tuple eliminations
+let [<Literal>] suffixForTupleElementAssignmentTarget = "tupleElem"
```

Comment:
I would probably name this `$tupleElem` instead of `tupleElem` because the user can never name a `val` with a `$`.

Reply:
@TIHan, it works for me when double-ticked. The check is guarded with `IsCompilerGenerated`, so it should be fine? `outArg` does not use `$` either. Let me know if you insist and I will of course change it;).

Reply:
It's probably fine then. I just want to make sure. :) Thank you for your work on this.

Reply:
My pleasure.